AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/script_canvas_tools_qt.py

```diff
@@ -0,0 +1,448 @@
+    )
+    Report.result(Tests.parameter_created, result)
+
+def remove_script_event_parameter(self):
+    remove_param = self.asset_editor_row_container.findChild(QtWidgets.QFrame, "[0]").findChild(QtWidgets.QToolButton, "")
+    remove_param.click()
+    result = helper.wait_for_condition(
+        lambda: self.asset_editor_widget.findChild(QtWidgets.QFrame, "[0]") is None, WAIT_TIME_3
+    )
+    Report.result(Tests.parameter_removed, result)
+
+def add_empty_parameter_to_script_event(self, number_of_parameters):
+    """
+    Function for adding a new blank parameter to a script event
+
+    param self: the script calling this function
+    param number_of_parameters: the number of empty parameters to add
+
+    returns none
+    """
+    helper.wait_for_condition(
+        lambda: self.asset_editor_row_container.findChild(QtWidgets.QFrame, PARAMETERS_QT) is not None, WAIT_TIME_3)
+    parameters = self.asset_editor_row_container.findChild(QtWidgets.QFrame, PARAMETERS_QT)
+    add_parameter = parameters.findChild(QtWidgets.QToolButton, "")
+
+    for _ in range(number_of_parameters):
+        add_parameter.click()
+
+def get_script_event_parameter_name_text(self):
+    """
+    function for retrieving the name field of script event parameters
+
+    param self: the script calling this function
+
+    returns a container with all the parameters' editable name fields
+    """
+    parameter_names = self.asset_editor_row_container.findChildren(QtWidgets.QFrame, NAME_STRING)
+    name_fields = []
+    for parameter_name in parameter_names:
+        name_fields.append(parameter_name.findChild(QtWidgets.QLineEdit))
+
+    return name_fields
+
+def get_script_event_parameter_type_combobox(self):
+    """
+    function for retrieving the type field of script event parameters
+
+    param self: the script calling this function
+
+    returns a container with all the parameters' editable type combo boxes
+    """
+    parameter_types = self.asset_editor_row_container.findChildren(QtWidgets.QFrame, "Type")
+    type_combo_boxes =[]
+    for parameter_type in parameter_types:
+        type_combo_boxes.append(parameter_type.findChild(QtWidgets.QComboBox))
+
+    return type_combo_boxes
+
+
+def create_entity_with_sc_component_asset(entity_name, source_file, position = math.Vector3(512.0, 512.0, 32.0)):
+    """
+    function for creating a new entity in the scene w/ a script canvas component. Function also adds as
+    script canvas file to the script canvas component's source file property.
+
+    param entity_name: the name you want to assign the entity
+    param source_file: the path to  script canvas file to be added to the script canvas component
+    param position: the translation property of the new entity's transform
+
+    returns: the entity created by this function
+    """
+    sourcehandle = scriptcanvas.SourceHandleFromPath(source_file)
+
+    entity = hydra.Entity(entity_name)
```

Comment:
Aren't we supposed to move away from "hydra" to editor_python_test_tools.editor_entity_utils

Reply:
Carlos put the SC component manipulation stuff into the hydra utils file.